reference/blob.v1.yaml

```diff
@@ -37,17 +37,101 @@ tags:
       APIs intended for internal use by Tidepool.
   - name: Blobs
     description: >-
+
       List, create, and manage binary blobs.
 
       The content format may be anything, because the blobs are treated as opaque data by the service.
       Each blob has a `status`.
       If a blob is created without content, then its status is `created`.
       If the blob is created with content or has been updated to include content then its status is `available`.
+  - name: Device Logs
+    description: >-
+      Create and list device logs.
 
 paths:
+  '/v1/users/{userId}/device-logs':
+    parameters:
+      - $ref: ./common/parameters/tidepooluserid.yaml
+    post:
+      operationId: UploadDeviceLogs
+      summary: Upload new device logs as json
+      description: Uploads a new device logs json to the user's account. The `Digest` header must represent the MD5 hash of the json data content.
+      parameters:
+        - $ref: ./common/parameters/digestmd5.v1.yaml
+        - $ref: ./blob/parameters/logs-start-at.v1.yaml
+        - $ref: ./blob/parameters/logs-end-at.v1.yaml
+      requestBody:
+        description: array of JSON data
+        required: true
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                $ref: ./blob/models/device-log-content.v1.yaml
+            example:
```

Comment:
It might be better to move the example to the definition of (each) model. That way, any reference to the model also "inherits" the examples.